lib/cdo/graphics/certificate_image.rb

```diff
@@ -24,18 +28,22 @@ def create_certificate_image2(image_path, name, params={})
   background
 end
 
-def create_course_certificate_image(name, course=nil, sponsor=nil)
+def create_course_certificate_image(name, course=nil, sponsor=nil, course_title=nil)
   name = name.gsub(/@/,'\@')
   name = ' ' if name.empty?
 
-  if course == '20hours' || course == 'hoc' || course == '20-hour' || course == 'hourofcode' || course == 'starwars'
-    # only need to fill in name
-    image_file = (course == '20hours' || course == '20-hour') ? '20hours_certificate.jpg' : 'hour_of_code_certificate.jpg'
-    vertical_offset = course == '20hours' || course == '20-hour' ? -115 : -110
-    image = create_certificate_image2(pegasus_dir('sites.v3', 'code.org', 'public', 'images', image_file), name, y: vertical_offset)
+  course ||= ScriptInfo::HOC_NAME
```

Comment:
Worth noting this is a slight functionality change from @breville's https://github.com/code-dot-org/code-dot-org/pull/5693 from yesterday where the default no-course-specified would show a non-HoC blank script certificate, whereas this would show an Hour of Code certificate.